modules/oc-mirror-migration-process.adoc

```diff
@@ -0,0 +1,133 @@
+apiVersion: mirror.openshift.io/v1alpha2
+mirror:
+  platform:
+    channels:
+      - name: stable-4.17
+    graph: true
+  helm:
+    repositories:
+      - name: sbo
+        url: https://redhat-developer.github.io/service-binding-operator-helm-chart/
+  additionalImages:
+    - name: registry.redhat.io/ubi8/ubi:latest
+    - name: quay.io/openshifttest/hello-openshift@sha256:example_hash
+  operators:
+    - catalog: oci:///test/redhat-operator-index
+      packages:
+        - name: aws-load-balancer-operator
+storageConfig:  # REMOVE this field in v2
+  local:
+    path: /var/lib/oc-mirror
+----
++
+.Example output with oc-mirror plugin v2 configuration:
+[source,yaml]
+----
+kind: ImageSetConfiguration
+apiVersion: mirror.openshift.io/v2alpha1
+mirror:
+  platform:
+    channels:
+      - name: stable-4.17
+    graph: true
+  helm:
+    repositories:
+      - name: sbo
+        url: https://redhat-developer.github.io/service-binding-operator-helm-chart/
+  additionalImages:
+    - name: registry.redhat.io/ubi8/ubi:latest
+    - name: quay.io/openshifttest/hello-openshift@sha256:example_hash
+  operators:
+    - catalog: oci:///test/redhat-operator-index
+      packages:
+        - name: aws-load-balancer-operator
+----
+
+. Commands have changed to align with oc-mirror plugin v2's new workflow.
+
+.. For mirror-to-disk, run the following command:
++
+[source,terminal]
+----
+$ oc-mirror --config isc.yaml file://some-dir --v2
+----
+
+.. For disk-to-mirror, run the following command:
++
+[source,terminal]
+----
+$ oc-mirror --config isc.yaml --from file://some-dir docker://remote-registry --v2
+----
+
+.. For mirror-to-mirror, run the following command:
++
+[source,terminal]
+----
+$ oc-mirror --config isc.yaml --from file://some-dir docker://remote-registry --v2
+----
++
+[NOTE]
+====
+`--workspace` is now required for mirror-to-mirror operation.
+====
+
+. In v2, mirrored content is stored in different locations. You must check the cluster-resources directory inside their working directory for IDMS, ITMS, `CatalogSource`, and `ClusterCatalog` using the following command :
++
+[source,terminal]
+----
+$ ls <workspace>/cluster-resources/
+----
+
+. oc-mirror plugin v2 replaces ICSP with IDMS/ITMS. Before applying the new configuration, delete the outdated ICSP entries to avoid conflicts using the following command:
++
+[source,terminal]
+----
+$ oc delete imagecontentsourcepolicy --all
+----
+
+. Remove images mirrored using oc-mirror plugin v1 using the following command:
++
+[source,terminal]
+----
+$ oc-mirror delete --config=delete-isc.yaml --generate --delete-v1-images --workspace file://some-dir docker://remote-registry --v2
+----
++
+[source,terminal]
+----
+oc-mirror delete --config=delete-isc.yaml --generate --delete-v1-images --workspace file://some-dir docker://remote-registry --v2
+----
++
+[IMPORTANT]
+====
+Operator index images are not deleted automatically. You must add them manually to `DeleteImageSetConfiguration`.
```

Comment:
Let' s tell them also the how and the why:
* maybe instead of saying index images , let' s talk about operator catalog images, which I think will be more identifiable
* v2 doesn' t delete catalog images because, in case the delete workflow was aiming at removing 1 or more operators belonging to that catalog from the registry, the catalog image should not be removed, because it would prevent the cluster, and OLM from functioning correctly
* If the user is sure that no more operators belonging to that catalog are on the registry or running in the cluster, they may force the deletion of the catalog image by explicitly adding it to the list of additionalImages in the deleteImageSetConfiguration

Reply:
Or say `catalog index image`